validphys2/src/validphys/config.py

```diff
@@ -1153,13 +1152,7 @@ def produce_nnfit_theory_covmat(
 
             f = user_covmat_fitting
 
-        @functools.wraps(f)
-        def res(*args, **kwargs):
-            return f(*args, **kwargs)
-
-        # Set this to get the same filename regardless of the action.
-        res.__name__ = "theory_covmat"
```

Comment:
I think it was, but the name of that function is not what decides the name of the stored .csv file and instead we have this 

https://github.com/NNPDF/nnpdf/blob/77c3343e3c71f917c89738a1970237a8bd8da395/validphys2/src/validphys/config.py#L817

Changing it now would break some old fits so probably not worth it